Accounts.Api/Program.cs

```diff
@@ -76,6 +77,9 @@
 
 var httpUrls = configuration.GetSection("HttpUrls");
 builder.Services.Configure<HttpUrls>(u => httpUrls.Bind(u));
+builder.Services.Configure<AccountValidOptions>(
+    u => configuration.GetSection(nameof(AccountValidOptions)).Bind(u)
```

Comment:
You can configure options without using `.Bind()`. Check the docs about options pattern in .net: https://learn.microsoft.com/en-us/dotnet/core/extensions/options